tests/keras_tests/feature_networks_tests/feature_networks/nested_networks/nested_test.py

```diff
@@ -87,9 +87,9 @@ def compare(self, quantized_model, float_model, input_x=None, quantization_info=
             else:
                 self.unit_test.assertFalse(isinstance(l, Functional) or isinstance(l, Sequential))
         if self.is_inner_functional:
-            num_layers = 9
+            num_layers = 8
```

Comment:
Why do we need to change this?
Is it something that is not stable in the test like the referencing to layers by index? If so, we might want to change the test so we won't need to update this everytime there is a change in the graph.

Reply:
This is because we're not using identity layers as before. So one layer was removed.